rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -881,6 +881,30 @@ void getActiveFederationCreationBlockHeight_afterRSKIP186AndNoValueInStorage_ret
 
     }
 
+    @Test
+    void testSetActiveFederationCreationBlockHeight() {
+
+        // Arrange
+
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        when(activations.isActive(ConsensusRule.RSKIP186)).thenReturn(true);
+
+        StorageAccessor storageAccessor = new InMemoryStorage();
+        FederationStorageProvider federationStorageProvider = new FederationStorageProviderImpl(storageAccessor);
+
+        // Act
+
+        federationStorageProvider.setActiveFederationCreationBlockHeight(3);
```

Comment:
I would store the value in an expected variable, aside from that LGTM

Reply:
agree!

Reply:
Updated.